src/status_im2/contexts/wallet/create_account/backup_recovery_phrase/view.cljs

```diff
@@ -0,0 +1,93 @@
+(ns status-im2.contexts.wallet.create-account.backup-recovery-phrase.view
+  (:require
+    [quo.core :as quo]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.create-account.backup-recovery-phrase.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+;; Temporary words
+(def words ["witch" "collapse" "practice" "feed" "shame" "open" "lion"
+            "collapse" "umbrella" "fabric" "sadness" "obligue"])
+
+(defn word-item
+  [item index _ increment]
+  [rn/view {:style style/word-item}
+   [quo/counter {:type :grey} (+ index increment)]
+   [quo/text {:style {:margin-left 4}} item]])
+
+(defn words-column
+  [words first?]
+  [rn/flat-list
+   {:style          {:padding-vertical 8}
+    :data           (if first? (subvec words 0 6) (subvec words 6))
+    :separator      [rn/view {:style {:height 12}}]
+    :render-fn      word-item
+    :render-data    (if first? 1 7)
+    :scroll-enabled false}])
+
+(defn step-item
+  [item index _ checked]
```

Comment:
item is just a label in this case? maybe it's a clearer name?

Reply:
`step-item` in every step component (checkbox + text) under "How to backup your recovery phrase"

Reply:
I meant the first argument here btw, "item", since the use of it is just a label it might be clearer of the use if you change it to that. `(defn step-item
  [label index _ checked]`